assets/scss/components/_dropdown.scss

```
@@ -0,0 +1,76 @@
+$dropdown-offset-right: 6px;
```

Pourquoi pas de `rem` ici ? Ça implique d'adapter plus bas avec `calc` par contre

```suggestion
$dropdown-offset-right: $length-6;
```